docs/advanced-topics/message-system/custom-messages.md

```diff
@@ -0,0 +1,53 @@
+---
+id: custom-messages
+title: Custom Messages 
+description: A brief explaination of Custom Message use in MLAPI coveing Named and Unnames messages.
+---
+
+If you don't want to use the MLAPI's messaging. You don't have to. You can use a thin layer called "Custom Messages" (these can be used in combination with [RPC messages](about-rpc.md) as well). Custom messages allows you to implement your own behaviour and add custom targeting etc. Custom messages are messages unbound to any game object.
+
+Custom messages comes in two forms, named and unnamed. Unnamed messages can be thought of as a single sending channel. A message sent has one receive handler, this is useful for building your own custom messaging system. If you want a completed messaging system, you can use named messages. The receiver registers one listen handler for each message type, and the sender can choose what type to send.
+
+## Unnamed Messages
+
+**Usage**
+
+```csharp
+private void Start()
+{
+    //Receiving
+    CustomMessagingManager.OnUnnamedMessage += ((senderClientId, stream) =>
```

Comment:
This will be a bit different in the next version. (correct for 0.1.0)

Reply:
This brings up a good point we should have a ticket to track doc updates between versions.